### ISSUE TITLE
[Feature Request] Sequential Failures

### ISSUE THREAD

**Issue by AngryBeaver** [AngryBeaver/beavers-crafting#54]
https://github.com/AngryBeaver/beavers-crafting/discussions/53

**Comment by AngryBeaver** [comment]
I do not think it is mainstream usage so currently i do not see this comeing . I will play with this idea and keep this feature request up. The key challenge about it is how to design a simple understandable ui to configure this.

**Comment by gantonaci** [comment]
In the Recipe configuration maybe have the header be:

Tests | Fails: 3 | Reset: <check mark> | Consume: <check mark> | +

**Comment by RawrKrake** [comment]
This feature is used in Kibbles Tasty'S Crafting, a relatively well known 5e Homebrew creator. I would love to see it integrated. I think the easiest implementation would be in the settings a box with "Reset failures at a success" or something like that.
